src/server.js

```diff
@@ -0,0 +1,11 @@
+import app from './app';
+import config from './config/config';
```

Comment:
We don't need this file. 

What `server.js`, `index.js` and `app.js` does is the same thing.

Reply:
Keep one of these files. I suggest `app.js` because it has almost all things.

Reply:
Keep one of these files. I suggest `app.js` because it has almost all things.